x-pack/plugins/infra/public/components/ml/anomaly_detection/flyout_home.tsx

```diff
@@ -104,25 +105,24 @@ export const FlyoutHome = (props: Props) => {
               />
             </h2>
           </EuiTitle>
+          <EuiTabs>
+            <EuiTab isSelected={tab === 'jobs'} onClick={() => setTab('jobs')}>
+              {i18n.translate('xpack.infra.ml.anomalyFlyout.jobsTabLabel', {
+                defaultMessage: 'Jobs',
+              })}
+            </EuiTab>
+            <EuiTab
+              isSelected={tab === 'anomalies'}
+              onClick={() => setTab('anomalies')}
+              data-test-subj="anomalyFlyoutAnomaliesTab"
+            >
+              {i18n.translate('xpack.infra.ml.anomalyFlyout.anomaliesTabLabel', {
+                defaultMessage: 'Anomalies',
+              })}
+            </EuiTab>
```

Comment:
Up to you, just a suggestion:

```suggestion
      {[
        {
          id: 'jobs',
          label: i18n.translate('xpack.infra.ml.anomalyFlyout.jobsTabLabel', { defaultMessage: 'Jobs' }),
        },
        {
          id: 'anomalies',
          label: i18n.translate('xpack.infra.ml.anomalyFlyout.anomaliesTabLabel', { defaultMessage: 'Anomalies' }),
        },
      ].map(({id, label}) => (
        <EuiTab
          key={id}
          isSelected={tab === id}
          onClick={() => setTab(id)}
          data-test-subj={`${id}AnomalyFlyout`}
        >
          {label}
        </EuiTab>
      ))}
```

You can also move the array definition outside the component.

Reply:
I think it's a good suggestion if we add more tabs in the future, for 2 tabs I will leave as it is